inference-engine/tests/functional/plugin/shared/src/behavior/stress_tests.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "behavior/stress_tests.hpp"
+#include "ngraph_functions/subgraph_builders.hpp"
+
+#define BUG_ALLOCATION_LIMIT 10000
+
+namespace LayerTestsDefinitions {
+
+    std::string MultipleAllocations::getTestCaseName(testing::TestParamInfo<LayerTestsUtils::TargetDevice> obj) {
+        return "targetDevice=" + obj.param;
+    }
+
+    void MultipleAllocations::SetUp() {
+        targetDevice = this->GetParam();
+        function = ngraph::builder::subgraph::makeSplitConvConcat();
+    }
+
+    TEST_P(MultipleAllocations, InferWorksCorrectAfter9999Allocations) {
+        SKIP_IF_CURRENT_TEST_IS_DISABLED()
+
+        ConfigurePlugin();
+
+        InferenceEngine::CNNNetwork cnnNet(function);
+        auto ie = PluginCache::get().ie();
+        for (int i = 0; i < BUG_ALLOCATION_LIMIT; ++i) {
+            std::cout << "\rLoadNetwork(): " << i << std::flush;
```

Comment:
Should we print this message 10k times?

Reply:
`\r` is a carriage return character. So, I think its Ok